Modules/Core/Common/include/itkPointSetBase.h

```diff
@@ -119,6 +119,13 @@ class ITK_TEMPLATE_EXPORT PointSetBase : public DataObject
   void
   Initialize() override;
 
+  /** Point dimension. The dimension of a point is fixed at compile-time. */
+  static constexpr unsigned int
+  GetPointDimension()
+  {
+    return PointDimension;
+  }
+
```

Comment:
This member function seems redundant, as `PointDimension` is already a public member. Are `static constexpr` public data members (like `PointDimension`) not made available already, by SWIG?

Reply:
@N-Dekker correct -- Python only has access to the run-time methods.

Reply:
Thanks Matt. It's a pity though, if SWIG cannot wrap them automatically. So just to be sure, I asked the question at swig/discussions as well: 

- https://github.com/swig/swig/discussions/3088

Reply:
On a high level, as a runtime language, Python has access to runtime information.

Reply:
The reply from William S Fulton (@wsfulton) at https://github.com/swig/swig/discussions/3088#discussioncomment-11550037 suggests that it _is_ possible to directly get the value of a static constexpr data member of a template class, in Python.  (So then the `GetPointDimension()` member function that you added should not be necessary.) However, I haven't yet been able to figure out how to make it work. 🤷 To be continued, hopefully...